ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/ontologies/views/dialogs/EdamUploadDialog.java

```diff
@@ -0,0 +1,219 @@
+package org.iplantc.de.admin.desktop.client.ontologies.views.dialogs;
```

Comment:
I could look into it.  For quick testing, it was easier to duplicate since `SimpleFileUploadDialog` had an extra service call to check if the file existed already in irods and I think the success/failed responses were different.